tests/fixtures/blockchainApplicationsManage.js

```diff
@@ -1,63 +1,81 @@
 const blockchainApplicationsManage = [
   {
-    name: 'Lisk',
+    chainName: 'Lisk',
     chainID: '00000001',
     state: 'active',
-    apis: [{ rest: 'http://165.227.246.146:9901', rpc: 'ws://165.227.246.146:9901/rpc-v3' }],
+    serviceURLs: [{ rest: 'http://165.227.246.146:9901', rpc: 'ws://165.227.246.146:9901/rpc-v3' }],
     address: 'lsk24cd35u49jd8szo3pnsqe5dsxwrnazyqqqg5eu',
+    logo: {
+      png: 'http://www.lisk.png'
+    },
     lastUpdated: 123456789,
     lastCertificateHeight: 9000,
     isDefault: true,
   },
   {
-    name: 'Colecti',
+    chainName: 'Colecti',
     chainID: 'mi34vyyd12g2lkf0rza1irws',
     state: 'active',
-    apis: [{ rest: 'https://service.colecti.com', rpc: 'wss://service.colecti.com' }],
+    serviceURLs: [{ rest: 'https://service.colecti.com', rpc: 'wss://service.colecti.com' }],
```

Comment:
@Balanced02 Please update the `rpc` property here to `ws` as well as other cases where it's used